MAINTAINERS.md

```diff
@@ -1,21 +1,25 @@
 # Maintainers
 
-MAINTAINERS:
+Code for this project is reviewed by any other member of the team before it is merged into the project. Team members include:
```

Comment:
Do we want a least one engineer to approve tech changes, at least one designer to approve design changes, etc.?Or keep it as anyone can approve any change?

Reply:
This is a good question. I was thinking that this doc could stay high-level, so that we could make agreements on the team for reviewing types of PRs and to broadly trust the group. For example ADRs reviewers are written on the ticket.

We could go into more depth here and outline specific PR types and who should review them. This would mean we would be obligated to update this if we change a workflow. 

We could update this to sat that **another** member instead of **any**:

> Code for this project is reviewed by another other member of the team before it is merged into the project. 

Or be more prescriptive:

> Code for this project is reviewed by another other member of the team before it is merged into the project. The required reviewers depend on the type of work. For a pull request that is a code change, at least one engineer reviews a code PR. The reviewers for ADRs are listed in the ticket for that ADR. 

Thoughts on either approach?

Reply:
There's also [CODEOWNERS](https://github.blog/2017-07-06-introducing-code-owners/) we could leverage. I'd love to be auto added as a reviewer if code touches CSS.

Reply:
Also wondering if we need to split this into disciplines. I don't wanna appear in both eng and design. But it might be nice for newcomers to know that someone's contributions aren't limited to one discipline. We might consider just listing everyone and their roles in 1 list. 

- [Firstname] [Lastname], [github username], [role]

Maybe there are other things we might include — such as affiliation, timezone, notes about relevant skills/interests, etc.